css-view-transitions-2/Overview.bs

```diff
@@ -515,6 +553,29 @@ When capturing the old or new state for an element, perform the following steps
 	Note: This is written in a monkey-patch manner, and will be merged into the algorithm once the L1 spec graduates.
 </div>
 
+## Addition to pending transition operations ## {#additions-to-pending-transition-operation}
```

Comment:
We need a similar block for "skip the view transition algorithm". To add a step to the end of that algorithm which invokes "[=outbound post-capture steps=]".

Reply:
Added a reaction to `finished promise`.